src/components/lyne-button/styles/lyne-button.component.scss

```diff
@@ -115,6 +114,11 @@
   user-select: none;
 }
 
+.button__label, .button__icon {
```

Comment:
Looking the final HTML, I don't see transitions, neither on the button or the label / icon

In addition, differences between "normal" state and "hover" look a bit weird: the button becomes wider and bumps a bit to the top. But I don't know how are they intended to be, I think this is a topic for Manuel.

Reply:
did you run "npm install" before testing? because I use newly added design tokens.

Reply:
ok! now it looks good :)